app/src/main/java/gov/hhs/cdc/trustedintermediary/wrappers/HttpClient.java

```diff
@@ -0,0 +1,10 @@
+package gov.hhs.cdc.trustedintermediary.wrappers;
+
+import java.io.IOException;
+import java.util.Map;
+
+public interface HttpClient {
+    String post(String path, Map<String, String> headerMap, String body) throws IOException;
+
+    String requestToken(String url, Map<String, String> headerMap, String body) throws IOException;
```

Comment:
I would make this method called `get` since `requestToken` isn't a concern of a generic HTTP client.  This means that the `ApacheClient`'s will need to be updated too because it implements this interface.

```suggestion
    String get(String url, Map<String, String> headerMap, String body) throws IOException;
```

Reply:
Good call!